docs/dev_tools/third_party.md

```diff
@@ -93,6 +94,23 @@ Please download and install it.
 	
 Make and Eclipse exports on POSIX-like operating systems targeting Nordic devices require the `srec_cat` executable from the [sRecord](http://srecord.sourceforge.net) package. It may be available from your package manager (such as apt-get or Brew). 
 
+### GNU ARM Eclipse (managed CDT projects) with GNU ARM Embedded Toolchain
+
+The [GNU ARM Eclipse](http://gnuarmeclipse.github.io) exporter generates ready to run managed CDT projects, and strives to do it as accurately as possible, considering the large number of explicit configuration options GNU ARM Eclipse uses. 
+
+Managed projects are projects that do not need manually created `make` files, but generate them automatically from a detailed description, which includes the list of source folders, include folders, preprocessor symbols and compiler command line options.
+
+The main advantage of providing all these details to Eclipse is that it can create a very accurate internal representation of the project, in order to correctly show which parts of the code are used, and pop up tooltips with the actual definition for most variables/functions.
```

Comment:
I would like to split this sentence into two or clarify it a little because one could read it as providing details creates pop-up tooltips or correctly shows pop-up tooltips.